mimic/test/dummy.py

```diff
@@ -93,36 +96,118 @@ def resource(self):
+                 url="https://api.external.example.com:8080",
+                 publicURL=None, internalURL=None, adminURL=None,
+                 versionInfoURL=None, versionListURL=None,
+                 type_id=u"example", enabled=False, uuid=uuid.uuid4(),
+                 tenantid_alias="%tenant_id%"
+                 ):
+        """
+        Create an :obj:`ExampleEndpointTemplate`.
+
+        :param text_type name: name of the service provided, e.g Cloud Files.
+        :param text_type region: region the service is provided in, e.g ORD.
+        :param text_type version: version of the service, e.g v1.
+        :param text_type url: basic URL of the service in the region.
+        :param text_type publicURL: public URL of the service in the region.
+        :param text_type internalURL: internal URL of the service in
+            the region.
+        :param text_type adminURL: administrative URL for the service in
+            the region.
+        :param text_type versionInfoURL: URL to get the version information
+            of the service.
+        :param text_type versionListURL: URL to get the list of supported
+            versions by the service.
+        :param text_type type_id: service type, e.g object-store
+        :param boolean enabled: whether or not the service is enabled
+            for all users. Services can be disabled for all tenants but still
+            be enabled on a per-tenant basis.
+        :param text_type uuid: unique ID for the endpoint within the service.
+        :param text_type tenantid_alias: by default the system uses the text
+            '%tenant_id%' for what to replace in the URLs with the tenantid.
+            This value allows the service adminstrator to use a different
+            textual value. Note: This is not presently used by Mimic which
+            just appends the tenant-id for internally hosted services, and
+            simply uses the URLs as is for externally hosted services.
+        """
+        self.id_key = uuid
+        self.region_key = region
+        self.type_key = type_id
         self.name_key = name
-        self.regions_and_versions = regions_and_versions
+        self.enabled_key = enabled
+        self.publicURL = publicURL if publicURL is not None else url
+        self.internalURL = internalURL if internalURL is not None else url
+        self.adminURL = adminURL if adminURL is not None else url
+        self.tenantAlias = tenantid_alias
+        self.versionId = version
+        self.versionInfo = (versionInfoURL
+                            if versionInfoURL is not None
+                            else url + '/versionInfo')
+        self.versionList = (versionListURL
+                            if versionListURL is not None
+                            else url + '/versions')
+
+
+def make_example_internal_api(case, response_message="default message",
+                              regions_and_versions=[('ORD', 'v1')]):
```

Comment:
@derwolfe thanks for the reminder; I've got that fixed in my next PR. I can back port it here.